mypy/semanal_typeargs.py

```diff
@@ -182,7 +186,7 @@ def validate_args(
                             is_error = True
                             self.fail(
                                 message_registry.INVALID_TYPEVAR_AS_TYPEARG.format(arg.name, name),
-                                ctx,
+                                ctx,  # TODO: can we use arg as the context
```

Comment:
what this means

Reply:
the current error points to the type application, it would be more appropriate to point to the arg